recipes-bsp/u-boot/files/0033-iot2050-Enable-eMMC-RPMB-support-and-OPTee-driver.patch

```diff
@@ -0,0 +1,47 @@
+From 963a046af89befe18cce1a208e5cba10d36216ee Mon Sep 17 00:00:00 2001
+From: Baocheng Su <baocheng.su@siemens.com>
+Date: Tue, 27 Sep 2022 10:21:48 +0800
+Subject: [PATCH 33/33] iot2050: Enable eMMC RPMB support and OPTee driver
+
+By enabling the eMMC RPMB support and the OPTee driver, UEFI secure boot
+can take advantage of the RPMB based secure storage provided by OPTee.
+
+This also prepares the RPMB otp key programming via U-Boot by enabling
+the `CONFIG_CMD_OPTEE_RPMB`.
+
+For security concerns, by default disable the RPMB operation in the
+`mmc` command.
+
+Signed-off-by: Baocheng Su <baocheng.su@siemens.com>
+---
+ configs/iot2050_pg1_defconfig | 4 ++++
+ configs/iot2050_pg2_defconfig | 4 ++++
+ 2 files changed, 8 insertions(+)
+
+diff --git a/configs/iot2050_pg1_defconfig b/configs/iot2050_pg1_defconfig
+index 0b9b3270f8..de53743f32 100644
+--- a/configs/iot2050_pg1_defconfig
++++ b/configs/iot2050_pg1_defconfig
+@@ -140,3 +140,7 @@ CONFIG_WDT=y
+ CONFIG_WDT_K3_RTI=y
+ CONFIG_WDT_K3_RTI_LOAD_FW=y
+ CONFIG_OF_LIBFDT_OVERLAY=y
++CONFIG_TEE=y
++CONFIG_OPTEE=y
++CONFIG_SUPPORT_EMMC_RPMB=y
++# CONFIG_CMD_MMC_RPMB is not set
```

Comment:
Do we want to expose this to PG1?

Reply:
I think no harm for PG1. The advantage is that we could keep a unified baseline for both variants.